.github/workflows/nightly_build_push.yml

```diff
@@ -35,11 +35,11 @@ jobs:
 
       - name: Build Project
         run: |
-          cargo build --release
+          cargo build --features short-prefix
```

Comment:
since we are calling this nightly build it might be confusing to use short prefix bc some people might try to use the docker image in prod. let's maybe rename to test build or smth

Reply:
sorry missed that.